### DISCUSSION TITLE
2.1.0 Released with updates for fine-tuning

### DISCUSSION THREAD

**Discussion by abdulfatir** [amazon-science/chronos-forecasting#390] [Announcements]
## 🚀 What's New?

- Chronos-2 can now be deployed on AWS via SageMaker JumpStart. Please check [this notebook](https://github.com/amazon-science/chronos-forecasting/blob/main/notebooks/deploy-chronos-to-amazon-sagemaker.ipynb) for details.
- [Scaled Dot Product Attention (SDPA)](https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html) is now used as the default attention implementation in Chronos-2. If you need to use the previous eager implementation, please load the model with `Chronos2Pipeline.from_pretrained(..., attn_implementation="eager")`.
- `predict_df` support has been added for older Chronos and Chronos-Bolt models. Now, all models (Chronos-2, Chronos-Bolt, Chronos) provide a unified pandas dataframe API. Note: Only Chronos-2 supports multivariate and covariate-informed forecasting.
- `Chronos2Pipeline.embed` has been added, enabling users to extract embeddings from the _last layer_ of the Chronos-2 encoder. 

## 🐛 Bug Fixes

- Fixed issues related to past-only covariates use during Chronos-2 fine-tuning. **If you're fine-tuning Chronos-2 models, we strongly recommend upgrading to `chronos-forecasting==2.1.0`**.
- Fixed issue related to multiple workers on windows.

## All Changes
* Fix pandas install instruction  by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/326
* Scope down GitHub token permissions by @AdnaneKhan in https://github.com/amazon-science/chronos-forecasting/pull/328
* Update Chronos-2 notebook install instructions by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/333
* Add logging steps in fine-tuning example by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/334
* [chronos-2] add support for SDPA by @kashif in https://github.com/amazon-science/chronos-forecasting/pull/331
* Treat notebooks as documentation in linguist by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/338
* Count notebooks as python in language stats by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/347
* Move SageMaker notebook to a new path by @shchur in https://github.com/amazon-science/chronos-forecasting/pull/348
* Revert "Move SageMaker notebook to a new path" by @shchur in https://github.com/amazon-science/chronos-forecasting/pull/349
* fix validate_and_prepare_single_dict_task by separating "past only" and "future known" cov keys by @HarvestStars in https://github.com/amazon-science/chronos-forecasting/pull/344
* Add Chronos-2 SageMaker notebook by @shchur in https://github.com/amazon-science/chronos-forecasting/pull/350
* Update model link to Hugging Face by @shchur in https://github.com/amazon-science/chronos-forecasting/pull/353
* Remove logo from README by @shchur in https://github.com/amazon-science/chronos-forecasting/pull/355
* Update Notebooks and README by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/356
* Add FutureWarning for CloudFront by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/357
* Relax `transformers` lower bound to >=4.41 by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/364
* Set `num_workers=0` in Chronos-2 test data loader by @IliasAarab in https://github.com/amazon-science/chronos-forecasting/pull/365
* Update long horizon heuristic for Chronos-Bolt by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/366
* Add predict_df support for Chronos and Chronos-Bolt by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/371
* Add Chronos2Pipeline.embed by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/361
* Mask past-only covariates during loss computation by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/379
* Chronos-2: Change default fine-tuning learning rate and remove experimental label by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/381
* Update version to 2.1.0rc1 for pre-release by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/383
* Update torch dependency version to >=2.2,<3 by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/388
* Bump version to 2.1.0 by @abdulfatir in https://github.com/amazon-science/chronos-forecasting/pull/389

## New Contributors
* @AdnaneKhan made their first contribution in https://github.com/amazon-science/chronos-forecasting/pull/328
* @kashif made their first contribution in https://github.com/amazon-science/chronos-forecasting/pull/331
* @HarvestStars made their first contribution in https://github.com/amazon-science/chronos-forecasting/pull/344
* @IliasAarab made their first contribution in https://github.com/amazon-science/chronos-forecasting/pull/365

**Full Changelog**: https://github.com/amazon-science/chronos-forecasting/compare/v2.0.1...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/amazon-science/chronos-forecasting/releases/tag/v2.1.0'>2.1.0</a>.</em>
